libs/ui-components/src/components/form/LabelsField.tsx

```diff
@@ -15,10 +15,16 @@ type LabelsFieldProps = {
   onChangeCallback?: (newLabels: FlightCtlLabel[], hasErrors: boolean) => void;
 };
 
-const LabelsField: React.FC<LabelsFieldProps> = ({ name, onChangeCallback, addButtonText, isEditable }) => {
+const LabelsField: React.FC<LabelsFieldProps> = ({
+  name,
+  onChangeCallback,
+  addButtonText,
+  isEditable: isEditableProp,
```

Comment:
```suggestion
  isEditable = true,
```

then we dont need the other changes below I guess

Reply:
True, changed!